charts/nri-bundle/README.md

```diff
@@ -98,81 +55,72 @@ In the following example you can see how to monitor a Redis integration with aut
+| global.affinity | object | `{}` | Sets pod/node affinities |
+| global.cluster | string | `""` | The cluster name for the Kubernetes cluster. |
+| global.containerSecurityContext | object | `{}` | Sets security context (at container level) |
+| global.customAttributes | object | `{}` | Adds extra attributes to the cluster and all the metrics emitted to the backend |
+| global.customSecretInsightsKey | string | `""` | Key in the Secret object where the insights key is stored |
+| global.customSecretLicenseKey | string | `""` | Key in the Secret object where the license key is stored |
+| global.customSecretName | string | `""` | Name of the Secret object where the license key is stored |
+| global.dnsConfig | object | `{}` | Sets pod's dnsConfig |
+| global.fargate | bool | false | Must be set to `true` when deploying in an EKS Fargate environment |
+| global.hostNetwork | bool | false | Sets pod's hostNetwork |
+| global.images.pullSecrets | list | `[]` | Set secrets to be able to fetch images |
+| global.images.registry | string | `""` | Changes the registry where to get the images. Useful when there is an internal image cache/proxy |
+| global.insightsKey | string | `""` | The license key for your New Relic Account. This will be preferred configuration option if both `insightsKey` and `customSecret` are specified. |
+| global.labels | object | `{}` | Additional labels for chart objects |
+| global.licenseKey | string | `""` | The license key for your New Relic Account. This will be preferred configuration option if both `licenseKey` and `customSecret` are specified. |
+| global.lowDataMode | bool | false | Reduces number of metrics sent in order to reduce costs |
+| global.nodeSelector | object | `{}` | Sets pod's node selector |
+| global.nrStaging | bool | false | Send the metrics to the staging backend. Requires a valid staging license key |
+| global.podSecurityContext | object | `{}` | Sets security context (at pod level) |
+| global.priorityClassName | string | `""` | Sets pod's priorityClassName |
+| global.privileged | bool | false | In each integration it has different behavior. See [Further information](#values-managed-globally-3) but all aims to send less metrics to the backend to try to save costs | |
+| global.proxy | string | `""` | Configures the integration to send all HTTP/HTTPS request through the proxy in that URL. The URL should have a standard format like `https://user:password@hostname:port` |
+| global.serviceAccount.annotations | object | `{}` | Add these annotations to the service account we create |
+| global.serviceAccount.create | string | `nil` | Configures if the service account should be created or not |
+| global.serviceAccount.name | string | `nil` | Change the name of the service account. This is honored if you disable on this cahrt the creation of the service account so you can use your own |
```

Comment:
```suggestion
| global.serviceAccount.name | string | `nil` | Change the name of the service account. This is honored if you disable on this chart the creation of the service account so you can use your own |
```

Reply:
Solved in [#4f2332e](https://github.com/newrelic/helm-charts/pull/795/commits/4f2332e6fc57c5e9842ddb8e958681644a537814)